tests/starwars/test_dsl.py

```diff
@@ -372,11 +369,11 @@ def test_root_fields_aliased(ds, client):
 
 
 def test_operation_name(ds):
-    query = dsl_gql(GetHeroName=DSLQuery(ds.Query.hero.select(ds.Character.name),))
+    query = dsl_gql(DSLQuery(ds.Query.hero.select(ds.Character.name),))
 
     assert (
         print_ast(query)
-        == """query GetHeroName {
+        == """{
```

Comment:
`test_operation_name` probably intended to test presence of removed part.
Probably test can be removed or library fixed?

Reply:
Good catch, thanks, FINV.